com.unity.cinemachine/Editor/Samples/SampleDependencyImporter.cs

```diff
@@ -170,6 +183,72 @@ static bool PromptUserImportDependencyConfirmation(List<string> dependencies)
                     "Import samples without their dependencies");
             }
         }
+
+#if CINEMACHINE_URP || CINEMACHINE_HDRP
+        static readonly string[] k_MaterialFolders = { "Materials", "Cameron/Model" };
+        static void ConvertMaterials(IEnumerable<string> folders)
+        {
+            foreach (var folder in folders)
+            {
+                foreach (var materialFolder in k_MaterialFolders)
+                {
+                    var materialDir = new DirectoryInfo(folder + "/" + materialFolder);
+                    if (!materialDir.Exists)
+                        continue;
+                    var materialInfos = materialDir.GetFiles("*.mat");
+                    foreach (var matInfo in materialInfos)
+                    {
+                        var localPath = matInfo.FullName[Application.dataPath.Length..];
+                        var material = AssetDatabase.LoadAssetAtPath<Material>("Assets/" + localPath);
+
+                     MaterialUpgrader.Upgrade(material, 
+    #if CINEMACHINE_URP
+                         new UnityEditor.Rendering.Universal.StandardUpgrader(material.shader.name), 
+    #elif CINEMACHINE_HDRP
+                         UnityEditor.Rendering.HighDefinition.MaterialUpgradeHelper.GetHDRPMaterialUpgraders(),
+    #endif
+                         MaterialUpgrader.UpgradeFlags.None);
+                    }
+                }
+            }
+        }
+
+        static void FixPrefabs(IEnumerable<string> folders)
+        {
+            foreach (var folder in folders)
+            {
+                var hdrpPath = folder + "/~HDRP/";
+    #if CINEMACHINE_HDRP
```

Comment:
You need to be more diligent about testing your code.  This does not execute in builtin.

Reply:
Good catch! Fixed.

Reply:
Tested with built-in, urp, hdrp. HDRP folder is cleaned up in all cases.